app/models/user.py

```diff
@@ -48,6 +49,7 @@ class User(db.Model, UserMixin, ModelMixin):
     )
     is_deleted: orm.Mapped[bool] = orm.mapped_column(default=False)
     role: orm.Mapped[str] = orm.mapped_column(sa.String(32), default=UserRole.user.value)
+    language: orm.Mapped[str] = orm.mapped_column(sa.String(5), default=UserPreferredLanguage.ua.value)
```

Comment:
Add `server_default` to `orm.mapped_column` params. `default` works only for new instances in db. `server_default` fills empty cells of already created instances

Reply:
And please fix the migration file, and make sure it also has server default value set